app/[locale]/[workspaceid]/chat/page.tsx

```diff
@@ -42,7 +42,7 @@ export default function ChatPage() {
 
           <div className="flex grow flex-col items-center justify-center" />
 
-          <div className="w-[300px] pb-8 sm:w-[400px] md:w-[500px] lg:w-[660px] xl:w-[800px]">
```

Comment:
300px on mobile is too narrow; it creates a very small text area, changing it to 100% with px-4 padding.